mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -758,37 +761,42 @@ func testAccMongoDBAtlasClusterAWSConfigdWithLabels(projectID, name, backupEnabl
 	`, projectID, name, backupEnabled, tier, region, labelsConf)
 }
 
-func testAccMongoDBAtlasClusterPrivateEndpointLinkConfig(projectID, providerName, region, vpcID, subnetID, securityGroupID string) string {
+func testAccMongoDBAtlasClusterPrivateEndpointLinkConfig(awsAccessKey, awsSecretKey, projectID, providerName, region, vpcID, subnetID, securityGroupID string) string {
 	return fmt.Sprintf(`
+		provider "aws" {
+			region     = "us-east-1"
+			access_key = "%[1]s"
+			secret_key = "%[2]s"
+		}
 		resource "mongodbatlas_private_endpoint" "test" {
-			project_id    = "%[1]s"
-			provider_name = "%[2]s"
-			region        = "%[3]s"
+			project_id    = "%[3]s"
+			provider_name = "%[4]s"
+			region        = "%[5]s"
 		}
 		resource "aws_vpc_endpoint" "ptfe_service" {
-			vpc_id             = "%[4]s"
+			vpc_id             = "%[6]s"
 			service_name       = "${mongodbatlas_private_endpoint.test.endpoint_service_name}"
 			vpc_endpoint_type  = "Interface"
-			subnet_ids         = ["%[5]s"]
-			security_group_ids = ["%[6]s"]
+			subnet_ids         = ["%[7]s"]
+			security_group_ids = ["%[8]s"]
 		}
 		resource "mongodbatlas_private_endpoint_interface_link" "test" {
 			project_id            = "${mongodbatlas_private_endpoint.test.project_id}"
 			private_link_id       = "${mongodbatlas_private_endpoint.test.private_link_id}"
 			interface_endpoint_id = "${aws_vpc_endpoint.ptfe_service.id}"
 		}
 		resource "mongodbatlas_cluster" "my_cluster" {
-		  project_id             = "%[1]s"
+		  project_id             = "%[3]s"
 		  name                   = "my-cluster-test"
```

Comment:
could you add a random name for the cluster name, you can add `fmt.Sprintf("test-acc-%s", acctest.RandString(10))`

Reply:
done